.github/workflows/env-loader-ci.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Environment value loader CI
+
+on:
+  pull_request:
```

Comment:
I understand release workflows being separate but for CI can we simplify and just have one workflow that executes all tests? Right now tools may be nicely separated but what if they say use a shared library and it gets modified, this sort of separation will just add headache later. Let's just have a single "unit tests" workflow that runs all Go tests.